### PR TITLE
docs(guide): add Kubescape-cli guide.

### PR DESCRIPTION
**Pull request by morethancertified**
Renamed "Use Cases" to "Guides" and added the Kubescape CLI Blog.

### CODE REVIEW

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+```export KUBECONFIG=/etc/rancher/k3s/k3s.yaml```
+
+Verify the cluster is up:
+
+![image](01-cluster-up.png)
+
+Forward the port to 4000 since the Flask server will use 5000. Feel free to modify this if you wish. 
+
+## Deploy a Pod
+For this deployment, we will use a very simple Flask container.
```
Comment (slashben) [inline docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:28]: I think it is not Flask anymore

**Review by slashben — DISMISSED** [review]

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+Alright, there are definitely some issues here. With some research, you’ll notice that 1.21 is not the latest version of NGINX. Upgrade that to the latest version:
+
+```
+nginx:latest
+```
+After you have made the modification and saved the file, rebuild the container and re-apply the manifest:
+```
+kubectl delete -f nginx-app.yml && kubectl apply -f nginx-app.yml
+```
+Finally, run `kubescape scan nginx:latest` to see the results. The results shown in the image are current as of the writing of this post, so yours may differ:
```
Comment (craigbox) [inline docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:240]:
```suggestion
Finally, run `kubescape scan nginx:latest` to see the results. The results shown in the image are current as of the writing of this guide, so yours may differ:
```

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+![image](11-image-scan-2.png)
+
+Several vulnerabilities have been remediated! Unfortunately, not all of the vulnerabilities, but that's the nature of security! Your individual results may vary due to the number of open vulnerabilities for NGINX when you’re going through this guide.
```
Comment (craigbox) [inline docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:244]:
```suggestion
Several vulnerabilities have been remediated! Unfortunately, not all of them, but that's the nature of security, and why you need a tool such as Kubescape. Your individual results will vary due to the current version and number of open vulnerabilities in NGINX when you’re going through this guide.
```

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+```
+Finally, run `kubescape scan nginx:latest` to see the results. The results shown in the image are current as of the writing of this post, so yours may differ:
+
+![image](11-image-scan-2.png)
+
+Several vulnerabilities have been remediated! Unfortunately, not all of the vulnerabilities, but that's the nature of security! Your individual results may vary due to the number of open vulnerabilities for NGINX when you’re going through this guide. 
+
+## Conclusion
+
+In this guide, you have used Kubescape to scan a basic Kubernetes cluster for vulnerabilities, researched those vulnerabilities, and remediated them with a very straightforward process thanks to the tools and documentation provided by ARMO. Kubescape can also scan Helm manifests, Kustomize directories, and more. While this guide only scratches the surface, you can leverage Kubescape CLI to harden your clusters to meet any security requirements you may have.
```
Comment (craigbox) [inline docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:248]:
```suggestion
In this guide, you have used Kubescape to scan a basic Kubernetes cluster for vulnerabilities, researched those vulnerabilities, and remediated them with a very straightforward process. Kubescape can also scan Helm manifests, Kustomize directories, and more. While this guide only scratches the surface, you can leverage Kubescape's CLI to harden your clusters to meet your own security requirements.
```

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+```
+#### Verify
+```
+kubectl get pods -n app
+```
+![image](02-get-pods.png)
+
+Great! Now that your cluster is set up and your pod is deployed, use Kubescape to harden it.
+
+## Use Kubescape CLI
```
[the remark this answers is not in the document]
Comment (craigbox) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:72]: (another nit: "Kubescape CLI" isn't a brand name, it's "the Kubescape CLI" or "the Kubescape command-line tool")

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+```
+kubescape scan
+```
+![image](03-kubescape-scan-1.png)
+
+Alright, the scan is finished. There’s a lot of output to go through. Using kubescape scan without any modifiers runs a scan using the MITRE and NSA frameworks. As you scroll down, you can see several of the controls scanned and their results. At the bottom, you’ll see your overall compliance score and the highest-stakes workloads:
+
+![image](04-first-scan-summary.png)
+
+This guide will not focus on the Traefik control failures but will address the Pod/nginx-app issue soon. First, knock out one of the "Network" control failures.
```
[the remark this answers is not in the document]
Comment (morethancertified) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:89]: This is something the users will see as it's meant to be followed. So even if we get a screenshot with it, the users will see it if they're following along.

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (morethancertified) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:72]: The subsections explained each piece of what we'd be doing, but I can add more. How is "Use the Kubescape CLI to Detect Control Failures"?

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+kubescape scan image nginx:1.21
+```
+![image](10-image-scan-1.png)
+
+Alright, there are definitely some issues here. With some research, you’ll notice that 1.21 is not the latest version of NGINX. Upgrade that to the latest version:
+
+```
+nginx:latest
+```
+After you have made the modification and saved the file, rebuild the container and re-apply the manifest:
```
[the remark this answers is not in the document]
Comment (morethancertified) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:236]: Another relic from the previous deployment. Sorry about that.

File: docs/docs/use-cases/Kubescape-cli/kubescape-cli.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (craigbox) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:89]: (Does K3s come with Traefik by default?)

[the remark this answers is not in the document]
Comment (morethancertified) [reply docs/docs/use-cases/Kubescape-cli/kubescape-cli.md:89]: Yep, it's the default configuration.

File: mkdocs.yml [hunk 8]
```diff
@@ -77,8 +77,9 @@ nav:
             - 'Frameworks': 'docs/frameworks-and-controls/frameworks.md'
             - 'Control library': 'docs/controls/index.md'
             - 'Configuring controls': 'docs/frameworks-and-controls/configuring-controls.md'
-        - Use Cases:
-            - 'Repo Scanning with GHA': 'docs/use-cases/Kubescape-gha/kubescape-gha.md'
+        - Guides:
+            - 'Configure checks on a GitHub repository': 'docs/use-cases/kubescape-gha/kubescape-gha.md'
+            - 'Harden a cluster': 'docs/use-cases/kubescape-cli/kubescape-cli.md'
```
[the remark this answers is not in the document]
Comment (morethancertified) [reply mkdocs.yml]: Done.

File: mkdocs.yml [hunk 9]
```diff
@@ -78,8 +78,8 @@ nav:
             - 'Control library': 'docs/controls/index.md'
             - 'Configuring controls': 'docs/frameworks-and-controls/configuring-controls.md'
         - Guides:
-            - 'Configure checks on a GitHub repository': 'docs/use-cases/kubescape-gha/kubescape-gha.md'
-            - 'Harden a cluster': 'docs/use-cases/kubescape-cli/kubescape-cli.md'
+            - 'Configure checks on a GitHub repository': 'docs/use-cases/kubescape-gha/index.md'
```
Comment (craigbox) [inline mkdocs.yml:81]: I think you also need to rename this file, which doesn't look like it was done in this commit.

Comment (morethancertified) [reply mkdocs.yml:80]: I updated both with a push several hours ago. I just synced the fork again to be sure. It looks right on my end.

**Comment by slashben** [comment]
Can we merge this @morethancertified ?

**Review by slashben — APPROVED** [review]

**Pull request merged**